### PR TITLE
Harpy Sprite Update 3

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

This will be my third sprite update pack for Harpies, with new sprites courtesy of StillxIcarus (Discord). These include owl wings, and forked tails. I've also personally made a new "Large Ear Tufts" variation. Many sprites have been split between Huescale and Whitescale variations, with Huescale offering greater variety of colors, while Whitescale offers better pure whites(A notable weakness of having only huescale options was that white winged harpies ended up a yellow-cream color). 

Additionally, I have updated all of the Harpy Hardsuit and Softsuit variations to now include **Wing Sleeves**, with some hardsuits even having personalized color schemes for their sleeves, depending on how important the wearer may be. Of course, all of the syndicate hardsuits have received their own unique schemes for each suit, including for the first time a species-specific Elite Syndicate Suit. 

image

image

Additionally, I am taking this as a convenient time to move a majority of the Harpy files to the core EE Directory, as I am taking direct responsibility for maintaining my creation here, rather than anywhere else. 

:cl:
- add: Harpy Update 3, New markings for Harpies, all new species-specific hardsuit & softsuit variations that feature wing sleeves.

### CODE REVIEW

**Review by OldDanceJacket — APPROVED** [review]

File: Resources/Locale/en-US/markings/harpy.ftl [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
-marking-HarpyWingFolded = Folded Wings
-marking-HarpyWingFolded-harpyfolded = Wings
+marking-HarpyWingDefaultWhitescale = Basic Wings (Whitescale)
+marking-HarpyWingDefaultWhitescale-whitescaleharpy = Wings
+
+marking-HarpyWingFoldedHuescale = Folded Wings (Huescale)
+marking-HarpyWingFoldedHuescale-huescaleharpyfolded = Wings
+
+marking-HarpyWingFoldedWhitescale = Folded Wings (Whitescale)
+marking-HarpyWingFoldedWhitescale-whitescaleharpyfolded = Wings
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/markings/harpy.ftl:11]:
```suggestion
marking-HarpyWingDefaultWhitescale = Basic Wings (Whitescale)
marking-HarpyWingDefaultWhitescale-whitescale_harpy = Wings

marking-HarpyWingFoldedHuescale = Folded Wings (Huescale)
marking-HarpyWingFoldedHuescale-huescale_harpy_folded = Wings

marking-HarpyWingFoldedWhitescale = Folded Wings (Whitescale)
marking-HarpyWingFoldedWhitescale-whitescale_harpy_folded = Wings
```

Why don't these and the one above not have underscores like the rest?

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Locale/en-US/markings/harpy.ftl (hunk 1 above)
Comment (VMSolidus) [reply Resources/Locale/en-US/markings/harpy.ftl:11]: They have to match the sprite states in the folder, would you like me to change those?

Comment (DEATHB4DEFEAT) [reply Resources/Locale/en-US/markings/harpy.ftl:11]: It would be nice for consistency, yes.

Comment (VMSolidus) [reply Resources/Locale/en-US/markings/harpy.ftl]: Done, my wrist hurts now.

Comment (DEATHB4DEFEAT) [reply Resources/Locale/en-US/markings/harpy.ftl]: What? Okay..

**Pull request merged**